app/src/main/java/com/example/sabonit/Account.java

```diff
@@ -13,29 +13,17 @@
  */
 public class Account
 {
-    /* ********* Constants: ********* */
-    // For validity phone number checking
-    private static int IMPLICIT_ISRAELI_PHONE_NUMBER_LENGTH = 9;
-    private static int ISRAELI_AREA_CODE = 13;
-    private static String ISRAELI_AREA_CODE_INFIX = "+972";
-
 
     /* ********* Attributes: ********* */
-    // Given as an address in the next pattern: street and building number, city ("Ramban 30, Jerusalem")
-    private String location;
-    // User's name, may can be extracted from the google acount
+    // User's name, may can be extracted from the google account
     private String name;
-    // User's phone number, may can be extracted from the google acount
-    private String phoneNumber;
```

Comment:
did you make sure you removed it from every place in the code?

Reply:
yes, the application runs fine, but it create problems we will fix now